BitbucketServerAutomation/Functions/Set-BBServerDefaultReviewer.ps1

```diff
@@ -0,0 +1,200 @@
+    #>
+    param(
+        [Parameter(Mandatory)]
+        [object]
+        # An object that defines what Bitbucket Server to connect to and the credentials to use when connecting. Use `New-BBServerConnection` to create connection objects.
+        $Connection,
+
+        [Parameter(Mandatory)]
+        [string]
+        # The key/ID that identifies the project. This is *not* the project name.
+        $ProjectKey,
+
+        [Parameter(Mandatory)]
+        [int]
+        # The ID of the default reviewer condition. Use `Get-BBServerDefaultReviewer` to get the ID of an existing condition.
+        $ID,
+
+        [int]
+        # The number of default reviewers that must approve a pull request.
+        $ApprovalCount,
+
+        [string]
+        # The name of a repository in the project.
+        $RepositoryName,
+
+        [ValidateSet('Any', 'Name', 'Pattern', 'Model')]
+        # The type of matching to do against the source branch.
+        $SourceBranchType,
+
+        [string]
+        # Specifies the value or pattern to match a source branch, if `-SourceBranchType` is not "Any".
+        $SourceBranchValue,
+
+        [ValidateSet('Any', 'Name', 'Pattern', 'Model')]
+        # The type of matching to do against the target branch.
+        $TargetBranchType,
+
+        [string]
+        # Specifies the value or pattern to match a target branch, if `-TargetBranchType` is not "Any".
+        $TargetBranchValue,
+
+        [object[]]
+        # Collection of objects representing the users to add to the default reviewer condition. Use `Get-BBServerUser` to get Bitbucket Server user objects.
+        $User
+    )
+
+    Set-StrictMode -Version 'Latest'
+    Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
+
+    if ($SourceBranchValue -and -not $SourceBranchType)
+    {
+        Write-Error -Message 'You must specify a "SourceBranchType" when giving a "SourceBranchValue".'
+        return
+    }
+
+    if ($TargetBranchValue -and -not $TargetBranchType)
+    {
+        Write-Error -Message 'You must specify a "TargetBranchType" when giving a "TargetBranchValue".'
+        return
+    }
+
+    $userProperties = @('name', 'emailAddress', 'id', 'displayName', 'active', 'slug', 'type')
+    foreach ($userObj in $User)
+    {
+        foreach ($property in $userProperties)
+        {
+            if (-not ($userObj | Get-Member -Name $property))
+            {
+                Write-Error -Message '"User" parameter''s argument does not have the expected properties. Use "Get-BBServerUser" to get user objects to pass to this cmdlet.' -ErrorAction $ErrorActionPreference
+                return
+            }
+        }
+    }
+
+    $notFoundErrorMessageSuffix = 'project "{0}"' -f $ProjectKey
+    $repositoryParam = @{ }
+    if ($RepositoryName)
+    {
+        $notFoundErrorMessageSuffix = 'repository "{0}" in project "{1}"' -f $RepositoryName, $ProjectKey

```

Comment:
Change to `$notFoundErrorMessageSuffix = 'repository "{0}" in {1}' -f $RepositoryName, $notFoundErrorMessageSuffix`